actions/user_actions.jsx

```diff
@@ -35,10 +35,10 @@ export function loadProfilesAndStatusesInChannel(channelId, page = 0, perPage =
     };
 }
 
-export function loadProfilesAndTeamMembers(page, perPage, teamId) {
+export function loadProfilesAndTeamMembers(page, perPage, teamId, sort = '', options) {
```

Comment:
we are not using the `sort` parameter in any call, do we need it?

Reply:
👍